web/app/js/components/Navigation.jsx

```diff
@@ -115,13 +114,17 @@ const styles = theme => {
     },
     linkerdNavLogoClose: {
       opacity: "0",
-      marginLeft: `-${navLogoWidth-theme.spacing.unit/2}px`,
+      marginLeft: `-${navLogoWidth+theme.spacing.unit/2}px`,
       transition: leavingFn(['margin', 'opacity']),
     },
     navMenuItem: {
       paddingLeft: `${contentPadding}px`,
       paddingRight: `${contentPadding}px`,
     },
+    shrinkIcon: {
+      fontSize: "20px",
```

Comment:
one nit: can we reduce this ever so slightly, to 18px?

Reply:
size updated, also bumped the padding to make the icons align through their centers:

![screen shot 2018-10-31 at 1 18 55 pm](https://user-images.githubusercontent.com/236915/47816196-abb7d680-dd0f-11e8-8f88-a47d9aa82d31.png)